source/adapters/level_zero/common.hpp

```diff
@@ -241,19 +248,6 @@ static const uint32_t UrL0QueueSyncNonBlocking = [] {
   return L0QueueSyncLockingModeValue;
 }();
 
-// Controls whether the L0 Adapter creates signal events for commands on
```

Comment:
why was this deleted? You have not changed the usage of that env and this is needed. Please keep this.